eng/templates/build.yml

```diff
@@ -39,4 +39,24 @@ jobs:
           inputs:
             SourceFolder: dist
             Contents: '**'
-            TargetFolder: $(Build.ArtifactStagingDirectory)
\ No newline at end of file
+            TargetFolder: $(Build.ArtifactStagingDirectory)
+
+        - task: ManualValidation@1
+          timeoutInMinutes: 1440 # task times out in 1 day
+          inputs:
+            notifyUsers: |
+              andystaples@microsoft.com
+            instructions: 'Approve publish of azure-functions-durable to PyPi'
+            onTimeout: 'reject'
+
+        - script: | 
+            pip install wheel
+            pip install twine
+
+        - task: TwineAuthenticate@1
+          displayName: Twine Authenticate
+          inputs:
+            pythonUploadServiceConnection: azure-functions-durable-pypi
+
+        - script: | 
+            python -m twine upload --config-file $(PYPIRC_PATH) dist/*
```

Comment:
You probably want to break this out into a different pipeline entirely. Keeping it in this job will cause these builds to fail each time you reject it. So only "released" runs will report as success.